scripts/packaging/win/build_installer.bat

```diff
@@ -1,37 +1,68 @@
 @echo on
 
-REM replace with one from environment
-set RELEASE_TYPE=dev
-
-aws s3 ls %STAGE_URL% --recursive
-
-heat.exe /?
-candle.exe /?
-light.exe /?
-
-
-REM heat.exe dir dist\\snow\\_internal ^
-REM   -gg ^
-REM   -cg SnowflakeCLIInternalFiles ^
-REM   -dr TESTFILEPRODUCTDIR ^
-REM   -var var.SnowflakeCLIInternalFiles ^
-REM   -sfrag ^
-REM   -o _internal.wxs
-REM
-REM candle.exe ^
-REM   -arch x64 ^
-REM   -dSnowflakeCLIVersion=%CLI_VERSION% ^
-REM   -dSnowflakeCLIInternalFiles=dist\\snow\\_internal ^
-REM   scripts\packaging\win\snowflake_cli.wxs ^
-REM   scripts\packaging\win\snowflake_cli_exitdlg.wxs ^
-REM   _internal.wxs
-REM
-REM light.exe ^
-REM   -ext WixUIExtension ^
-REM   -ext WixUtilExtension ^
-REM   -cultures:en-us ^
-REM   -loc scripts\packaging\win\snowflake_cli_en-us.wxl ^
-REM   -out snowflake-cli-%CLI_VERSION%-x86_64.msi ^
-REM   snowflake_cli.wixobj ^
-REM   snowflake_cli_exitdlg.wixobj ^
-REM   _internal.wixobj
+set PATH=C:\Program Files\7-Zip;C:\Users\jenkins\AppData\Local\Programs\Python\Python38;C:\Users\jenkins\AppData\Local\Programs\Python\Python38\Scripts;C:\Program Files (x86)\WiX Toolset v3.11\bin;%PATH%
+
+python.exe --version
+python.exe -c "import platform as p; print(f'{p.system()=}, {p.architecture()=}')"
+
+python.exe -m pip install hatch
+FOR /F "delims=" %%I IN ('hatch run packaging:win-build-version') DO SET CLI_VERSION=%%I
+FOR /F "delims=" %%I IN ('git rev-parse %svnRevision%') DO SET REVISION=%%I
+FOR /F "delims=" %%I IN ('echo %releaseType%') DO SET RELEASE_TYPE=%%I
+@echo off
+echo CLI_VERSION = %CLI_VERSION%
+echo REVISION = %REVISION%`
+echo RELEASE_TYPE = %RELEASE_TYPE%`
```

Comment:
This was for debug?

Reply:
We can say so. It's meant to simplify reading job output.